includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -2621,6 +2622,35 @@ public function cancel_authorization( $order ) {
 		];
 	}
 
+	/**
+	 * Create the shipping data array to send to Stripe when making a purchase.
+	 *
+	 * @param WC_Order $order The order that is being paid for.
+	 * @return array          The shipping data to send to Stripe.
+	 */
+	public function get_shipping_data_from_order( WC_Order $order ): array {
+		$object_to_parse = $order;
```

Comment:
Why do we introduce a local variable here? Can we use `$order` instead?

Reply:
I think this code was taken from draft by @dmvrtx , maybe he can comment it better

Reply:
Maybe it was for better readability, since $order is too generic.  While reviewing code, it is visually clearer where the param is referred, especially while scrolling through many other functions. Maybe that was the intention.

Reply:
Let's just use `$order` then.

Reply:
Thanks. I have changed it within this commit:
242cb56